www/layouts/home.html

```diff
@@ -114,7 +114,7 @@ <h2>Your Donation Makes a Difference</h2>
     </div>
   </div>
   {{ partial "ocw_news.html" . }}
-  {{ partial "home_testimonials.html" . }}
+  {{ partial "home_stories.html" . }}
```

Comment:
I'd suggest removing `home_` from the partial name so it can be a bit more generic. 
We did something similar in featured and new courses too ([here](https://github.com/mitodl/ocw-hugo-themes/pull/566#discussion_r838547416))